dashboards-observability/common/constants/explorer.ts

```diff
@@ -269,3 +269,14 @@ export enum ConfigChartOptionsEnum {
 
 export const CUSTOM_LABEL = 'customLabel';
 export const BREAKDOWNS = 'breakdowns';
+
+// Metrics constants
+export const UNITS_OF_MEASURE = [
+  'milliseconds (ms)',
```

Comment:
Milliseconds is no longer necessary; can you confirm @pjfitzgibbons?

Reply:
oh okay, i'll remove it